fixtures/integrations/jira/stubs/unexpected_user.json

```diff
@@ -0,0 +1,16 @@
+{
+  "active": true,
+  "avatarUrls": {
+    "24x24": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png",
+    "16x16": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png",
+    "48x48": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png",
+    "32x32": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png"
+  },
+  "displayName": "Saif Hakim",
+  "accountType": "atlassian",
+  "timeZone": "America/Los_Angeles",
+  "locale": "en_US",
+  "emailAddress": "",
+  "self": "https://example.atlassian.net/rest/api/2/user?accountId=012345:00000000-1111-2222-3333-444444444444",
+  "unexpectedId": "012345:00000000-1111-2222-3333-444444444444"
```

Comment:
This is used in `src/sentry/tests/sentry/integrations/jira/test_utils.py` in `test_unexpected_id`. I refactored `build_user_choice` because it was taking in a `user_id_field` that differed based on whether Jira Cloud or Jira Server was using it. Each has their own implementation of this now.